src/util/saveArticles.ts

```diff
@@ -402,9 +403,32 @@ async function treatVideo(mw: MediaWiki, dump: Dump, srcCache: KVS<boolean>, art
     }
 
     sourceEl.setAttribute('src', newUrl);
+
+    /* Scrape subtitle */
+    const trackEle = videoEl.querySelector('track');
+    if (trackEle) {
+        const { content, trackTitle } = await treatSubtitles(trackEle, downloader, webUrlHost, mw);
+        const article = new ZimArticle({ url: `${trackTitle}.vtt`, mimeType: 'text/vtt', data: content, ns: 'I' });
+        zimCreator.addArticle(article);
+        trackEle.setAttribute('src', `${getRelativeFilePath(articleId, trackTitle, 'I')}.vtt`);
```

Comment:
are you sure should we put subtitles into `I` namespace? ;)

Reply:
If there is no namespace for the resource, default is "I", do you suggest something different?

Reply:
For all the content and image download, if no namespace is provided the default is "I", so I followed that, but as I can see for other module files(css, js) default convention is "-", so I will change it to that.

Reply:
I just mean that I was wondering why we put the subtitles in one namespace next to the images while subtitles are the text by nature ;)

Reply:
Yes, I am aware that the rest of the text content file(js, css) uses "-" as namespace, but how they produce links are also different [here]. So, that was the reason I found using "I" namespace best for this usecase.(https://github.com/openzim/mwoffliner/blob/9ea0be2c9fed9463c9f380627e608af882c6cddf/src/util/misc.ts#208)